### ISSUE TITLE
Prestashop Productcomments - Receive a notification when a new review is received

### ISSUE THREAD

**Issue by photinia73** [PrestaShop/PrestaShop#17837]
Hello!

If it possible to email to webmaster new comment for moderation from the module Productcomments?
>Edited by Khouloud

it is not possible to receive any notification when a new review is added.
But, it could be a good improvement.
The improvement could add an option similar to this
![image](https://user-images.githubusercontent.com/16067358/75442482-e0698b80-595f-11ea-9691-68f70b57cb0f.png)

**Comment by florine2623** [comment]
Hello @photinia73 ,
What is your issue exactly ?

Thanks ! Please give us feedback.

**Comment by photinia73** [comment]
I installed the PS module Productcomments.
In the configuration I choosed "All reviews must be verified by a moderator" is ON
But how I will know, that someone left comment, if any notification or letter I didn`t received. I tested this module and left the review....
So is it any possibility to send the notification that new comment is waiting to be verified?
Thanks!

**Comment by khouloudbelguith** [comment]
Hi @photinia73,

>In the configuration I choosed "All reviews must be verified by a moderator" is ON

When a customer adds a new review, this review will not automatically added to the FO, the employee should approve this review first.
So, a new line is added in the section: "Reviews waiting for approval"
![image](https://user-images.githubusercontent.com/16067358/75428945-472f7a80-5949-11ea-88d8-c2dc0a1f1d08.png)

In this time, the employee should approve or delete this review.

- If the review approved => in the FO, it will be displayed
- If the review deleted => in the FO, it will not be displayed

Thanks to check & feedback.

**Comment by photinia73** [comment]
I understand this. But how will I know, that this new review wait? Any
notification not come. I need to check every time this menu in BO to verify
if the new review appear and wait, this is not comfortably (( may be in a
future PS will add such notification function (like about new order)
Thanks!

чт, 27 лют. 2020, 11:12 користувач khouloudbelguith <
notifications@github.com> пише:

**Comment by khouloudbelguith** [comment]
@photinia73, no, it is not possible to receive any notification when a new review is added.
But, it could be a good improvement.
The improvement could add an option similar to this
![image](https://user-images.githubusercontent.com/16067358/75442482-e0698b80-595f-11ea-9691-68f70b57cb0f.png)

Ping @PrestaShop/prestashop-product-team what do you think?

Thanks!

**Comment by photinia73** [comment]
It will be great! Will wait )) Thanks

чт, 27 лют. 2020, 13:52 користувач khouloudbelguith <
notifications@github.com> пише:

**Comment by photinia73** [comment]
Some else!
After Google check there is next problem in the Productcomments module
![image](https://user-images.githubusercontent.com/50954519/75672844-46b91b80-5c8a-11ea-97d5-533711e25a1f.png)

**Comment by LouiseBonnard** [comment]
I agree with @khouloudbelguith, it would be nice to have this option in the **Advanced Parameters > Administration** page of the back office. Another idea to complete the idea would be to add this option to the **Favicon Notifications BO** module.
